src/workflow/CompareCommitToRevisionWorkflow.php

```diff
@@ -51,7 +51,7 @@ public function getCommandHelp() {
   }
 
   public function requiresWorkingCopy() {
-    return true;
+    return $this->useLocalEnlistment = $this->getArgument('use-local-enlistment');
```

Comment:
Is this short-hand for:

```
$this->useLocalEnlistment = $this->getArgument('use-local-enlistment');
return $this->useLocalEnlistment;
```

?

A bit unclear why we're setting a property in a function that seems to be checking whether a working copy is needed.

Why is this not sufficient:
```
public function requiresWorkingCopy() {
    return $this->getArgument('use-local-enlistment') == true;
}
```

Reply:
:(. copy/paste error.   fixed now.  Good catch.